storage/watchable_store_test.go

```diff
@@ -15,47 +15,216 @@
 package storage
 
 import (
+	"bytes"
+	"fmt"
 	"os"
+	"strings"
 	"testing"
+
+	"github.com/coreos/etcd/storage/storagepb"
+)
+
+var (
+	testN      = 100
+	testKey    = []byte("Foo")
+	testValue  = []byte("Bar")
+	testKeys   = make([][]byte, testN)
+	testValues = make([][]byte, testN)
 )
 
-func TestWatch(t *testing.T) {
```

Comment:
Can we keep pervious test unchanged in the pull request? We just need to add a test for newWatchingToEventMap.

If we want to refactor the test, we should do it in a separate pull request.